.github/workflows/build-and-test.yaml

```diff
@@ -36,7 +36,14 @@ jobs:
           target-packages: ${{ steps.get-self-packages.outputs.self-packages }}
           build-depends-repos: build_depends.repos
 
+      - name: Check file existence
+        id: check_files
+        uses: andstor/file-existence-action@v1
+        with:
+          files: lcov/total_coverage.info,coveragepy/.coverage
+
       - name: Upload coverage to Codecov
+        if: steps.check_files.outputs.files_exists == 'true'
```

Comment:
Don't you need to add this to build-and-test-pr.yaml?

Reply:
Currently, the pr workflow uses the `fail_ci_if_error: false` option to avoid a failure. However, I think it would be better to use 
a file check script instead of this option.
So, I will change the script.

Reply:
0862981